packages/fetch-extension/src/pages-new/register/mnemonic/recover-mnemonic.tsx

```diff
@@ -142,33 +142,35 @@ export const RecoverMnemonicMainPage: FunctionComponent<{
             leftImage={keyIcon}
             heading={"Use a seed phrase or a private key"}
           />
-          <Card
-            leftImageStyle={{
-              width: "32px",
-              height: "32px",
-            }}
-            style={{
-              backgroundColor: "rgba(255,255,255,0.1)",
-              height: "78px",
-              fontSize: "14px",
-              padding: "18px",
-            }}
-            onClick={(e: any) => {
-              e.preventDefault();
-              setSelectedCard(AccountSetupType.CONNECT_HARDWARE);
-              registerConfig.setType(TypeRecoverMnemonic);
-              analyticsStore.logEvent("Import account started", {
-                registerType: "seed",
-              });
-            }}
-            leftImage={require("@assets/svg/wireframe/hardware.svg")}
-            heading={"Connect hardware wallet"}
-            subheading={"Requires bluetooth access to pair"}
-            subheadingStyle={{
-              fontSize: "14px",
-              opacity: 0.6,
-            }}
-          />
+          {uiConfigStore.platform != "firefox" && (
+            <Card
+              leftImageStyle={{
+                width: "32px",
+                height: "32px",
+              }}
+              style={{
+                backgroundColor: "rgba(255,255,255,0.1)",
+                height: "78px",
+                fontSize: "14px",
+                padding: "18px",
+              }}
+              onClick={(e: any) => {
+                e.preventDefault();
+                setSelectedCard(AccountSetupType.CONNECT_HARDWARE);
+                registerConfig.setType(TypeRecoverMnemonic);
+                analyticsStore.logEvent("Import account started", {
+                  registerType: "seed",
+                });
+              }}
+              leftImage={require("@assets/svg/wireframe/hardware.svg")}
+              heading={"Connect hardware wallet"}
+              subheading={"Requires bluetooth access to pair"}
```

Comment:
@AbbasAliLokhandwala We need to update this label to indicate that the extension connects to the ledger via cable instead of bluetooth.